pmm-app/tests/ia/pages/alertRulesPage.js

```diff
@@ -1,5 +1,5 @@
 const { I } = inject();
-const rulesNameCell = (ruleName) => `//td[1][div[text()="${ruleName}"]]`;
+const rulesNameCell = (ruleName) => `//td[1][div/span[text()="${ruleName}"]]`;
```

Comment:
can't use locate?

Reply:
no, I use this func in building another locators. For some reason, strange exceptions occur. So using this for now